src/main/js/wiki/components/ActionHeader.tsx

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import injectSheet from "react-jss";
+import ActionButton from "./ActionButton";
+import PageNameForm from "./PageNameForm";
+import { withRouter } from "react-router-dom";
+import classNames from "classnames";
+import BranchDropdown from "./BranchDropdown";
+import { Branch } from "../../repository/types/repositoryDto";
+
+const styles = {
+  flexbox: {
+    justifyContent: "space-between",
+    flexDirection: "column"
+  },
+  actions: {
+    display: "flex",
+    flexDirection: "row",
+    flexWrap: "wrap",
+    justifyContent: "space-between",
+    alignItems: "flex-start",
+    marginBottom: "1em"
+  }
+};
+
+type Props = {
+  wiki: any;
+  path: any;
+  history: any;
+  classes: any;
+  inSettings: boolean;
+  inEdit: boolean;
+  pushBranchStateFunction: (branchName: string, pagePath: string) => void;
+  branch: string;
+  branches: Branch[];
+};
+
+type State = {
+  showCreateForm: boolean;
+  showMoveForm: boolean;
```

Comment:
the move and delete form can be deleted from the properties

Reply:
- [x] done